implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OtlpMeterRegistryTest.java

```diff
@@ -129,6 +144,22 @@ void timeGauge() {
                     + "    time_unix_nano: 1000000\n" + "    as_double: 0.024\n" + "  }\n" + "}\n");
     }
 
+    @Test
+    void httpHeaders() throws Throwable {
```

Comment:
Not crucial but we have an `@Issue` annotation that could be used here to connect the issue number to the test, something like this:

```suggestion
    @Test
    @Issue("#5577")
    void httpHeaders() throws Throwable {
```